### ISSUE TITLE
Colorbrewer colormap orientations are inconsistent

### ISSUE THREAD

**Issue by briochemc** [tsipkens/cmap#4]
Just noticed that out of the diverging colorbrewer colormaps listed here

<img width=33% alt="Screen Shot 2022-03-31 at 3 34 42 pm" src="https://user-images.githubusercontent.com/4486578/160977384-b83b6d86-804e-48d1-af85-cb002ebffe95.png">

Only the first and last ones are oriented in the same way that https://colorbrewer2.org/#type=diverging is. In other words `rdylgn`, `piyg`, and `prgn` should be reversed to match the original colormaps. Note that I have not checked other colormaps!

I think colorbrewer chose these orientations because we traditionally expect red-ish colors to represent the most positive values (i.e., at the top/right/end of the colormap). 

---

That being said, colorbrewer's naming convention itself is not consistent, e.g., `RdBu` starts with blue (at the bottom) while `PuOr` starts with orange...

<img width="116" alt="Screen Shot 2022-03-31 at 3 48 20 pm" src="https://user-images.githubusercontent.com/4486578/160978706-5e7d6002-5e55-48ea-a9b3-c9bdd7191a84.png"> <img width="116" alt="Screen Shot 2022-03-31 at 3 48 13 pm" src="https://user-images.githubusercontent.com/4486578/160978711-b8ae5a94-29ab-4ae6-9192-c2f6f4e0e6f7.png">

So maybe it is worth correcting colorbrewer's inconsistency here and only having names that match the color order? I.e., `RdBu` would go from red to blue. An ideal solution is probably to also allow for `BuRd` in this case, which would go blue to red. What do you think?

**Comment by tsipkens** [comment]
Good catch. I have opted instead to match the diverging colormap order to their names rather than the original orders. This required reversing of the `RdBu` colormap. The colormaps can be easily reversed using MATLAB's built-in `flipud(...)` function.

Fixed in the master branch.

**Comment by briochemc** [comment]
> The colormaps can be easily reversed using MATLAB's built-in flipud(...) 

I know! I meant we could easily add in convenience functions that go, e.g., 

```matlab
function cm = pubu(n, opt_interp)
    cm = flipup(bupu(n, opt_interp))
end
```

Given your decision to opt for "the colormap follows the name order", this would be nice IMHO :)

**Comment by tsipkens** [comment]
Reopened while I consider this.
